package.json

```diff
@@ -7,12 +7,14 @@
     "build:dev": "webpack && npm run start:dev",
     "transpile": "npx babel ./server --out-dir dist",
     "start": "NODE_ENV=production ./node_modules/pm2/bin/pm2 start dist/bin/www.js",
-    "start:dev": "node_modules/nodemon/bin/nodemon.js  --exec babel-node server/bin/www.js",
+    "dev": "NODE_ENV=development npm-run-all --parallel watch:server watch:build",
     "monitor": "./node_modules/pm2/bin/pm2 monit",
     "stop": "./node_modules/pm2/bin/pm2 stop all && ./node_modules/pm2/bin/pm2 delete all",
     "logs": "./node_modules/pm2/bin/pm2 logs --lines 100",
     "restart": "NODE_ENV=production ./node_modules/pm2/bin/pm2 stop all && ./node_modules/pm2/bin/pm2 delete all && ./node_modules/pm2/bin/pm2 start dist/bin/www.js",
-    "node": "DEBUG=* node ./dist/bin/www.js"
+    "node": "DEBUG=* node ./dist/bin/www.js",
+    "watch:build": "webpack --watch",
+    "watch:server": "nodemon --ignore ./public --exec babel-node server/bin/www.js --watch \"./server\""
```

Comment:
what do you think about swapping these command names, so `build:watch` and `serve:watch`? The command seems to be `build` or `serve` and the "watch" part is a modification of that command. For instance, we already have a `build` command so `build:watch` modifies that.

Reply:
I updated the names and went from start:dev to just dev.
[f67f383](https://github.com/AMP-SCZ/dpdash/commit/f67f383401bf0105196fd976a7c6a4fce63d6f22)